packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -11,15 +11,32 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		}
+	},
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build-and-test": "npm run build && npm run test",
 		"build:compile": "fluid-build . --task compile",
-		"build:test": "tsc --project ./e2e-tests/tsconfig.json",
+		"build:esnext": "tsc --project ./tsconfig.json",
+		"build:test": "npm run build:test:esm && npm run build:test:cjs",
+		"build:test:cjs": "fluid-tsc commonjs --project ./e2e-tests/tsconfig.cjs.json",
+		"build:test:esm": "tsc --project ./e2e-tests/tsconfig.json",
```

Comment:
Note that 'build:test' builds the e2e-tests, but there are additional unit tests that are built comingled with the source.

Reply:
If it would be helpful to reorganize the test code, that would be fine. We could add `unit-tests` and `end-to-end-tests` directly under `src`, and move everything as appropriate. If it would make things cleaner here.

Reply:
The PR comment is just an FYI for the node16 team, since this conversion looks a bit different than the others.  (Ditto below).

We have a tech debt item filed about comingled test/src builds, but there's no need to do anything now.